package.json

```diff
@@ -14,7 +14,7 @@
     "@editorjs/list": "^1.6.0",
     "@editorjs/quote": "^2.4.0",
     "@lhci/cli": "^0.4.1",
-    "@saleor/sdk": "^0.3.0-2",
+    "@saleor/sdk": "git://github.com/mirumee/saleor-sdk.git#update-order-status-type",
```

Comment:
We should have only released sdk version on master

Reply:
@orzechdev it was feature branch, we know it :)